addons/mail/tests/test_mail_channel_as_guest.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo
+from odoo.tests import HttpCase
+
+@odoo.tests.tagged('-at_install', 'post_install')
+class TestMailGuestPages(HttpCase):
+    def test_mail_channel_as_guest(self):
+        """Checks that the invite page redirects to the channel and that all
+        modules load correctly on the welcome and channel page"""
+        channel = self.env['mail.channel'].create({
+            'name': 'Test channel',
```

Comment:
Probably a bit short. Shouldn't we specifically target public channels, ... ?

Reply:
The main use case is actually private channels (group created from "start a meeting"). But all channels are available through invitation link (except DM), so the type or visibility is not really a factor for the current test. We just want to have at least one test to ensure the page even loads at all, as it's doing a lot of custom booting code that will definitely be broken by changes in assets bundles if people don't know it exists.